web/src/redux/githubID/graphAuthConfig.js

```diff
@@ -0,0 +1,21 @@
+/* this is used by the (hopefully silent) OAuth flow to get a token to access Microsoft Graph's API assigning contacts 
+to project requests. It should not require editing. You can find the Azure project, where the Tenant and Client IDs are
+defined, here: https://portal.azure.com/#view/Microsoft_AAD_RegisteredApps/ApplicationMenuBlade/~/Overview/appId/5afdfc62-637b-41cf-b186-b2de816faaf9/isMSAApp~/false
+or look for 'Platform Services Registry' on BC Gov's Azure environment. */
+
+export const msalConfig = {
+  auth: {
+    clientId: '5afdfc62-637b-41cf-b186-b2de816faaf9',
```

Comment:
Let's access these values from environment variables set in `.env` instead. Since this React application was scaffolded with Create React App, we need to follow their way of doing this: https://create-react-app.dev/docs/adding-custom-environment-variables/

Your environment variables in `.env` should look like `REACT_APP_AZURE_CLIENT_ID` and `REACT_APP_AZURE_TENANT_ID`. The `REACT_APP_` prefix is necessary for any environment variables to show up in an app created with Create React App.